Neo4jClient.DataAnnotations/CypherFluentQueryExtensions.cs

```diff
@@ -380,7 +380,7 @@ public static ICypherFluentQuery Merge(this ICypherFluentQuery query, params Exp
         /// </summary>
         public static ICypherFluentQuery Create(this ICypherFluentQuery query, params Expression<Func<IPathBuilder, IPathExtent>>[] patternDescriptions)
         {
-            return WithPattern(query, out var pattern, PropertiesBuildStrategy.WithParams, patternDescriptions).Create(pattern);
+            return WithPattern(query, out var pattern, PropertiesBuildStrategy.WithParamsForValues, patternDescriptions).Create(pattern);
```

Comment:
I want to merge this PR now since Neo4jClient 4.0 is now available in stable version. However, can I know why you made this change here? 

I haven't been following all the latest changes with Neo4j of late, so I wouldn't know if perhaps you added this change because of an update in Neo4j.